db/change_cache.go

```diff
@@ -309,7 +309,7 @@ func (c *changeCache) waitForSequenceWithMissing(sequence uint64) {
 
 // Given a newly changed document (received from the tap feed), adds change entries to channels.
 // The JSON must be the raw document from the bucket, with the metadata and all.
-func (c *changeCache) DocChanged(docID string, docJSON []byte, seq uint64, vbNo uint16) {
+func (c *changeCache) DocChanged(docID string, docJSON []byte, seq uint64, vbNo uint16, dataType uint8) {
```

Comment:
That doesn't work, because MemcachedDataType isn't a single flag (uint), it's a set of flags (uint8).